content/blog/2025/q1-product-goals/index.md

```diff
@@ -0,0 +1,59 @@
+
+**In Q1 2025** we'll begin this expansion by deploying JupyterHubs and BinderHubs on JetStream 2, which will give communities access to publicly-funded computing resources. We will use this experience to decide whether it's sustainable for us to deploy on this and other publicly-owned infrastructure providers.
+
+## Enable enhanced community knowledge bases with Jupyter Book 2
+
+A key theme we aim to enable is **sharing** within and between community hubs. This is a critical part of the data science workflow because it allows people to collaborate on the same ideas, and build on top of one another's ideas. An early target for this is to facilitate lightweight sharing of computational content so that community members can learn from one another more effectively.
+
+**In Q1 2025** we want to help get [Jupyter Book 2's beta](https://next.jupyterbook.org) released, and provide an out-of-the-box configuration for our communities to use it with their hubs. This includes adding landing pages and better integration with JupyterHub via launch buttons to create a more seamless experience between documentation and interactive computing.
+
+## Enable sharing reproducible environments on a hub
+
+Another key aspect of sharing is **sharing the computational environment as well**. This would allow communities to not only sheir their content, but also live infrastructure that allows others to reproduce and interact with their work. We think that investing more time into imiproving and deploying [BinderHubs](https://binderhub.readthedocs.io/en/latest/) (the technology behind [mybinder.org](https://mybinder.org)) will help us learn more about how to make this a reality.
+
+**In Q1 2025** we plan to grow our capacity to deploy BinderHubs across multiple cloud providers. This will allow hub users to build their own Binder environments on the fly and make it possible to share these environments with others, enabling better reproducibility and collaboration within communities.
+
+## Give communities more visibility and control over their hub setup and costs
+
+Perhaps the biggest perceived risk to using cloud infrastructure is the possibility of **runaway costs**. Community leaders are often nervous that something unexpected will happen and they'll have to foot a giant bill at the end of the month. We think that reducing this risk is a key way to make cloud infrastructure safer and more useful for research and education communities.
+
+**In Q1 2025** we are aim to add more **visibility into hub usage** and **more controls over resources via quotas**. This will allow more fine control over resource budgets such as CPU, memory, and storage. We'll also work on breaking down **users into groups** for better resource management, giving communities more granular control over their infrastructure.
```

Comment:
```suggestion
**In Q1 2025** we are aim to add more **visibility into hub usage** and **more controls over resources via quotas**. This will allow more fine control over resource budgets such as CPU, memory, and storage. We'll also work on **assigning users to groups**, allowing communities greater control over resource allocation across large user bases.
```